src/imio/esign/browser/settings.py

```
@@ -39,6 +43,37 @@ def validate_vat_number(va_nb):
     return True
 
 
+def validate_external_watchers(tal_expression):
+    """Validate external watchers is a valid TAL expression with valid email addresses."""
+
+    if not tal_expression or not tal_expression.strip():
+        return True
+
+    try:
+        portal = api.portal.get()
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)